contentcuration/contentcuration/views/internal.py

```diff
@@ -672,6 +679,10 @@ def create_node(node_data, parent_node, sort_order):
         freeze_authoring_data=True,
         role_visibility=node_data.get('role') or roles.LEARNER,
     )
+    if len(tags) > 0:
+        node.tags = tags
```

Comment:
Oh cool, I didn't know we can assign to M2M rel'n managers directly!  I thought we needed to do something like `node.tags.add(*tags)` (append to set) or `node.tags.set(*tags)` (to overwrite any existing).

Learn something every day...